keyboards/narfkb/rules.mk

```
@@ -0,0 +1,11 @@
+SERIAL_DRIVER = vendor
+
+ENCODER_ENABLE = yes
+BACKLIGHT_ENABLE = no
+RGB_MATRIX_ENABLE = no
+RGBLIGHT_ENABLE = no
+MOUSEKEY_ENABLE = no
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+BOOTMAGIC_ENABLE = no
+LTO_ENABLE = no
```

Unless you have a good reason for these 3 features to be disabled, it's generally preferred that they're enabled.
```suggestion
BOOTMAGIC_ENABLE = yes
EXTRAKEY_ENABLE = yes
MOUSEKEY_ENABLE = yes
```